smoke-tests/apps/ClassicSdkWebInterop2x/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/ClassicSdkWebInteropTest.java

```diff
@@ -54,7 +54,7 @@ void doMostBasicTest() throws Exception {
         .isEqualTo("mydeviceosversion");
     assertThat(telemetry.rd.getProperties()).containsEntry("myattr1", "myvalue1");
     assertThat(telemetry.rd.getProperties()).containsEntry("myattr2", "myvalue2");
-    assertThat(telemetry.rd.getProperties()).hasSize(2);
+    assertThat(telemetry.rd.getProperties()).hasSize(3);
```

Comment:
can you add

```
assertThat(telemetry.rd.getProperties()).containsEntry("_MS.ProcessedByMetricExtractors", "True");
```

in these places also, nice to verify the exact contents of the properties